diff --git a/dist/index.mjs b/dist/index.mjs
index 85b4c0a516291a217183391316901c6c48dcd09a..52972c88958f5018fdfcac5d0e64c725a0b9fbdc 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -2196,7 +2196,21 @@ const validTitle = createEslintRule({
           }
         },
         patternProperties: {
-          [/^must(?:Not)?Match$/u.source]: {
+          mustMatch: {
+            oneOf: [
+              { type: "string" },
+              MatcherAndMessageSchema,
+              {
+                type: "object",
+                //@ts-ignore
+                propertyNames: { type: "string", enum: ["describe", "test", "it"] },
+                additionalProperties: {
+                  oneOf: [{ type: "string" }, MatcherAndMessageSchema]
+                }
+              }
+            ]
+          },
+          mustNotMatch: {
             oneOf: [
               { type: "string" },
               MatcherAndMessageSchema,
